pkg/adaptation/adaptation_suite_test.go

```diff
@@ -488,6 +488,13 @@ var _ = Describe("Plugin container creation adjustments", func() {
 		case "rlimit":
 			a.AddRlimit("nofile", 456, 123)
 
+		case "CDI device":
```

Comment:
nit: Seems strange that this is a space-separated string.

Reply:
Updated to use `"CDI-device"`.